# Raspberry Pi 5: raise OP-TEE DRAM0_SIZE from 8GB to 16GB so OP-TEE is aware
# of all DRAM on 16GB boards (safe on smaller). From the rpi5 reference port:
# https://github.com/jonasjuffinger/OP-TEE-on-the-RPi-5 (patches/optee_os.diff).
diff --git a/core/arch/arm/plat-rpi5/platform_config.h b/core/arch/arm/plat-rpi5/platform_config.h
index 913fe5a..7762455 100644
--- a/core/arch/arm/plat-rpi5/platform_config.h
+++ b/core/arch/arm/plat-rpi5/platform_config.h
@@ -17,6 +17,6 @@
 #define CONSOLE_UART_CLK_IN_HZ  0
 
 #define DRAM0_BASE		0x00000000
-#define DRAM0_SIZE		0x200000000
+#define DRAM0_SIZE		0x400000000
 
 #endif /* PLATFORM_CONFIG_H */
